content/Api/Blip.Api.Template.Services/Blip.Api.Template.Services.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.1</TargetFramework>
-    <ProjectGuid>sonarqube-guid-services</ProjectGuid>
+    <ProjectGuid>{8562C21A-DC12-4F70-AA93-C136563F2CD8}</ProjectGuid>
```

Comment:
whenever you build the project, or open in VS even, the placeholder gets replaced by the generated GUID. Revert back to the placeholder